packages/@holaplexui-playground/package.json

```
@@ -9,7 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@holaplex/ui-library-react": "*",
+    "@holaplex/ui-library-react": "^0.4.0",
```

```suggestion
    "@holaplex/ui-library-react": "*",
```
You can use star here. I tested it. Try clearing out all of your node_modules and reinstalling.